lib/skate_web/components/core_components.ex

```diff
@@ -27,4 +27,41 @@ defmodule SkateWeb.CoreComponents do
     <% end %>
     """
   end
+
+  defp banner_background_color("skate-local"), do: "gray"
+  defp banner_background_color("skate-dev"), do: "orange"
+  defp banner_background_color("skate-dev-blue"), do: "blue"
+  defp banner_background_color("skate-dev-green"), do: "green"
+  defp banner_background_color("skate-prod"), do: "red"
+  defp banner_background_color(_), do: "gray"
```

Comment:
nit/suggestion(non-blocking): do you think you could replace these with the [mbta colors from `tokens-2024`](https://github.com/mbta/skate/blob/main/assets/css/color/_tokens_2024.scss#L170-L180)?

It's fine if not, I'm considering a PR to move most of this to CSS and make our admin CSS better.

Reply:
Does [this](https://github.com/mbta/skate/pull/2594/commits/b76bf5a9b1a84db3a99c0b8cff99b7eaee3daeae) do what you were hoping? I feel like this is a partial move towards better CSS, but only partial.

Reply:
Great thanks!

One nit is that `skate-prod` should probably be `bus` because it's _possible_ for us in the future to have a `dev-red`, but this works for me as is.